tests/zfs-tests/tests/functional/slog/slog_014_pos.ksh

```diff
@@ -83,6 +83,9 @@ for type in "mirror" "raidz" "raidz2"; do
 		fi
 
 		log_must zpool online $TESTPOOL $VDIR/a
+		log_must zpool sync $TESTPOOL
+		log_must zpool wait -t resilver $TESTPOOL
+		log_must zpool wait -t scrub $TESTPOOL
```

Comment:
This will slow down the test a little but is a reasonable precaution.  Adding a `-f` to the `zpool create` above wouldn't be a bad idea either in case `$VDIR/a` isn't brought online and is left with a stale label.

Reply:
it is interest usecase where we we see resilvering first with next scrub - it was found by additional `zpool status` with many tries

Reply:
OK, I'm all for some additional sanity check to make sure the tests run as reliably as possible.